FoodWaste/Views/Calculator/RoundedViews.swift

```diff
@@ -94,6 +94,7 @@ struct RoundedWasteButton: View {
 struct RoundedFunctionButton: View {
     var function: FunctionParameter
     var buttonAction: () -> Void
+    var disableButton: () -> Bool = { false }
```

Comment:
this param can be changed to a Bool instead of function:
var disableButton: Bool = false